nemo/core/classes/exportable.py

```diff
@@ -128,7 +129,7 @@ def _export(
             # Set module mode
             with torch.onnx.select_model_mode_for_export(
                 self, training
-            ), torch.inference_mode(), torch.no_grad(), torch.jit.optimized_execution(True):
+            ), torch.inference_mode(), torch.no_grad(), torch.jit.optimized_execution(True), _jit_is_scripting():
```

Comment:
Hmm we don't use lightning during export, is this method safe ?

Reply:
Would the tests have confirmed if this is fine? The underlying code is simple to what is happening:

In Lightning:
```python
@contextmanager
def _jit_is_scripting() -> Generator:
    """Workaround for https://github.com/pytorch/pytorch/issues/67146."""
    LightningModule._jit_is_scripting = True
    try:
        yield
    finally:
        LightningModule._jit_is_scripting = False
```

Reply:
Hmm i dont think it would be caught in tests, lets keep it for now